test/resources/playbooks/docker/Dockerfile.j2

```diff
@@ -9,9 +9,9 @@ FROM {{ item.image }}
 ARG testarg=fail
 ENV envarg=$testarg
 
-RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get install -y python sudo bash ca-certificates && apt-get clean; \
-    elif [ $(command -v dnf) ]; then dnf makecache && dnf --assumeyes install python sudo python-devel python2-dnf bash && dnf clean all; \
-    elif [ $(command -v yum) ]; then yum makecache fast && yum install -y python sudo yum-plugin-ovl bash && sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf && yum clean all; \
-    elif [ $(command -v zypper) ]; then zypper refresh && zypper install -y python sudo bash python-xml && zypper clean -a; \
+RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get install -y python sudo bash ca-certificates iproute2 && apt-get clean; \
+    elif [ $(command -v dnf) ]; then dnf makecache && dnf --assumeyes install python sudo python-devel python*-dnf bash iproute && dnf clean all; \
```

Comment:
Was `python*-dnf` added to make this consistent with the other file? Good spot.

Reply:
Yes, the `python*-dnf` change in #1649 was not complete.

I checked the package names in `alpine`, `debian`, `centos`, `fedora`, `opensuse/leap`, `ubuntu` and `voidlinux`. The `alpine` one has the `ip` command of `busybox` but that seems to satisfy ansible expectations.

Reply:
Thanks for confirming, and spotting this extra issue in the first place!